pywinauto/keyboard.py

```diff
@@ -98,10 +98,19 @@
     send_keys('{^}a{^}c{%}') # type string "^a^c%" (Ctrl will not be pressed)
     send_keys('{{}ENTER{}}') # type string "{ENTER}" without pressing Enter key
 
+For Windows only, pywinauto defaults to sending a virtual key packet
+(VK_PACKET) for textual input.  For applications that do not handle VK_PACKET
+appropriately, the ``vk_packet`` option may be set to ``False``.  In this case
+pywinauto will attempt to send the virtual key code of the requested key.  This
+option only affects the behavior of keys matching [-=[]\;',./a-zA-Z0-9 ].  Note
+that upper and lower case are included for a-z.  Both reference the same
+virtual key and are included for convenience.

```

Comment:
Looks like a typo: "Both reference the same virtual key and are included for convenience." -> I'd suggest "Both reference the same virtual key for convenience."